draft-ietf-mls-protocol.md

```diff
@@ -1279,71 +1279,145 @@ Given these inputs, the derivation of secrets for an epoch
 proceeds as shown in the following diagram:
 
 ~~~~~
-                  init_secret_[n-1]
-                        |
-                        V
-   commit_secret -> KDF.Extract = joiner_secret
-                        |
-                        +--> Derive-Secret(., "welcome")
-                        |    = welcome_secret
-                        |
-                        V
-                  Derive-Secret(., "member")
-                        |
-                        V
-      PSK (or 0) -> KDF.Extract = member_secret
-                        |
-                        V
-                  Derive-Secret(., "epoch")
-                        |
-                        V
-GroupContext_[n] -> KDF.Extract = epoch_secret
-                        |
-                        +--> Derive-Secret(., <label>)
-                        |    = <secret>
-                        |
-                        V
-                  Derive-Secret(., "init")
-                        |
-                        V
-                  init_secret_[n]
+                   init_secret_[n-1]
+                         |
+                         V
+    commit_secret -> KDF.Extract = joiner_secret
+                         |
+                         V
+                   Derive-Secret(., "member")
+                         |
+                         V
+psk_secret (or 0) -> KDF.Extract = member_secret
+                         |
+                         +--> Derive-Secret(., "welcome")
+                         |    = welcome_secret
+                         |
+                         V
+                   Derive-Secret(., "epoch")
+                         |
+                         V
+ GroupContext_[n] -> KDF.Extract = epoch_secret
+                         |
+                         +--> Derive-Secret(., <label>)
+                         |    = <secret>
+                         |
+                         V
+                   Derive-Secret(., "init")
+                         |
+                         V
+                   init_secret_[n]
 ~~~~~
 
 A number of secrets are derived from the epoch secret for different purposes:
 
-| Secret                | Label         |
-|:----------------------|:--------------|
-| `sender_data_secret`  | "sender data" |
-| `handshake_secret`    | "handshake"   |
-| `application_secret`  | "app"         |
-| `exporter_secret`     | "exporter"    |
-| `confirmation_key`    | "confirm"     |
+| Secret                  | Label         |
+|:------------------------|:--------------|
+| `sender_data_secret`    | "sender data" |
+| `handshake_secret`      | "handshake"   |
+| `application_secret`    | "app"         |
+| `exporter_secret`       | "exporter"    |
+| `confirmation_key`      | "confirm"     |
+| `confirmation_key`      | "confirm"     |
```

Comment:
Seems like the `confirmation_key` got duplicated accidentally

```suggestion
| `confirmation_key`      | "confirm"     |
```

Reply:
Whoops you're right. For some reason I can't commit your suggestion, but I'll change the PR accordingly.